js/validation/index.js

```diff
@@ -100,5 +100,7 @@ function parseAndValidateCodemeta(showPopup) {
         }
     }
 
+    const expanded = await jsonld.expand(parsed);
+    doc = await jsonld.compact(expanded, CODEMETA_CONTEXT_URL);
```

Comment:
```suggestion
    doc = await jsonld.compact(parsed, CODEMETA_CONTEXT_URL);
```

this should be equivalent

Reply:
It is indeed!